### DISCUSSION TITLE
is it possible to customize the icon for each feed?

### DISCUSSION THREAD

**Discussion by coolbq** [FreshRSS/FreshRSS#3875] [Ideas]
When adding the channel to FreshRSS, it fetches the favicon from the source site. But after sometime, I found it will help if I can customize the the icons for different feed channel. This will help improving the reading experience. 

I tried to replace the generated favicons under the /data/favicons respectively. It worked for some days, but turned back to the blank icon today. I tried again but this time all failed. 

So do we have a way to customize the icons for FreshRSS now ?

**Comment by Alkarex** [comment]
Indeed, the favicons are regularly refreshed from the original Web site, and defining a custom favicon manually is not an option we have at the moment

**Comment by Alkarex** [comment]
Implemented in https://github.com/FreshRSS/FreshRSS/pull/7646 , tests welcome
